.docker/docker-compose.override.default.yml

```diff
@@ -25,6 +25,23 @@ services:
     volumes:
     - ../:/var/www/html:cached
 
+  traefik:
```

Comment:
Why it was placed inside of `docker-compose.override.default.yml` folder? I believe it should sit in the main `docker-compose.yml` file. The remaining looks good.

Reply:
Well... it was initially implemented in CW when I thought having a non-changed docker-compose.yml from docker4drupal project is a good idea :) Apparently it wasn't - but it was sitting in the override. Agree, will move to the main docker-compose.yml file.